airflow/task/standard_task_runner.py

```diff
@@ -286,8 +279,7 @@ def _read_task_logs(self, stream):
             if not line:
                 break
             self.log.info(
-                "Job %s: Subtask %s %s",
-                self._task_instance.job_id,
+                "Subtask %s %s",
```

Comment:
this does not have a clear meaning. do you know what it means? maybe we can clarify?

Reply:
Honestly: I don't even know what it meant before with the "Job: %s". That was first added way back in in #2835, and the linked Jira issue says:

> Currently, when the scheduler is outputting all subtask logs to the same stream, it is not easy to distinguish which logs come from which task. It would be nice if there were some convenient way to filter the logs from a given task. For example, putting the task id after the word 'Subtask'

So yeah, entirely pointless now that each task logs to it's own file, not via the scheduler!

Reply:
(I'm not going to remove the prefix entirely here in this PR, and I'm going to revisit logging wholesale for AIP-72)